### PR TITLE
Fix dimension-specific code in VoronoiCovarianceMeasure

### PR DESCRIPTION
**Pull request by rolanddenis**
# PR Description

This bug raises from the crash of `examples/geometry/surfaces/dvcm-2d-curvature` in Debug mode.

# Checklist

- [N/A] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [N/A] Doxygen documentation of the code completed (classes, methods, types, members...)
- [N/A] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/geometry/surfaces/estimation/VoronoiCovarianceMeasureOnDigitalSurface.ih [hunk 1]
```diff
@@ -127,7 +127,7 @@ VoronoiCovarianceMeasureOnDigitalSurface( ConstAlias< Surface > _surface,
         {
           Point p = *itPts;
           const EigenStructure& evcm = myPt2EigenStructure[ p ];
-          VectorN n = evcm.vectors.column( 2 );
+          VectorN n = evcm.vectors.column( Space::dimension-1 );
```
Comment (rolanddenis) [inline src/DGtal/geometry/surfaces/estimation/VoronoiCovarianceMeasureOnDigitalSurface.ih]: @JacquesOlivierLachaud I'm not sure to understand what is going on here ... can you confirm that it is what you meant ?

Comment (JacquesOlivierLachaud) [inline src/DGtal/geometry/surfaces/estimation/VoronoiCovarianceMeasureOnDigitalSurface.ih]: The VCM normal corresponds to the greatest eigenvalue of the VCM covariance. If the eigenvectors are sorted in increasing eigenvalues, then your fix seems correct to me.

Comment (rolanddenis) [inline src/DGtal/geometry/surfaces/estimation/VoronoiCovarianceMeasureOnDigitalSurface.ih]: Ok, thanks !

**Comment by JacquesOlivierLachaud** [comment]
Can you just pull the current master and solve the conflict in ChangeLog ?
Thanks

**Comment by rolanddenis** [comment]
Done but don't know what is wrong with AppVeyor...

**Comment by JacquesOlivierLachaud** [comment]
Le 17/03/2016 11:19, rolanddenis a écrit :
> Done but don't know what is wrong with AppVeyor...
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DGtal-team/DGtal/pull/1145#issuecomment-197804358>
> 

Don't know. @dcoeurjo any idea ?

**Comment by rolanddenis** [comment]
I have re-triggered the checks and it seems ok now :octocat:

**Comment by dcoeurjo** [comment]
Great. Merging

**Pull request merged**